optaplanner-examples/src/main/java/org/optaplanner/examples/conferencescheduling/persistence/ConferenceSchedulingImporter.java

```diff
@@ -0,0 +1,496 @@
+/*
+ * Copyright 2018 Red Hat, Inc. and/or its affiliates.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.optaplanner.examples.conferencescheduling.persistence;
+
+import java.io.IOException;
+import java.io.InputStream;
+import java.io.StringReader;
+import java.time.Instant;
+import java.time.LocalDateTime;
+import java.time.ZoneId;
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.HashMap;
+import java.util.HashSet;
+import java.util.List;
+import java.util.Map;
+import java.util.Set;
+import java.util.function.Function;
+import java.util.stream.Collectors;
+
+import javax.json.Json;
+import javax.json.JsonArray;
+import javax.json.JsonObject;
+import javax.json.JsonReader;
+
+import org.apache.commons.lang3.tuple.Pair;
+import org.optaplanner.examples.conferencescheduling.domain.ConferenceParametrization;
+import org.optaplanner.examples.conferencescheduling.domain.ConferenceSolution;
+import org.optaplanner.examples.conferencescheduling.domain.Room;
+import org.optaplanner.examples.conferencescheduling.domain.Speaker;
+import org.optaplanner.examples.conferencescheduling.domain.Talk;
+import org.optaplanner.examples.conferencescheduling.domain.TalkType;
+import org.optaplanner.examples.conferencescheduling.domain.Timeslot;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+public class ConferenceSchedulingImporter {
+
+    private static final Logger logger = LoggerFactory.getLogger(ConferenceSchedulingImporter.class);
+    private static final String zoneId = "Europe/Paris";
```

Comment:
Ow, this is a pain: devoxx poland and devoxx UK and devoxx morocco are in other timezones.

Options:

- Simply default it to the default locale's timezones, so `ZoneId.systemDefault()`.
-- There's a JVM -D system property to override this per java process.
-- That presumes that the conference organizer scheduler is in the same timezone as the conference itself. That's usually true, it's a good default.
-- this is good enough for phase 1.
- We make the zoneId user input, like the restBaseUrl, defaulted to system default.
- Or we somehow get it from the rest service somehow:
-- ask stephan for a rest api to know it?
-- Look at HTTP headers to figure out the servers's timezone?
--- This is often unreliable, especially now that everything's hosted in the cloud
--- There's a plan to host the UK and FR/BE stuff on the same server (simply different tenants), IIRC.
--